packages/mephisto-task/src/index.js

```diff
@@ -68,6 +69,14 @@ const useMephistoTask = function () {
     [state.agentId]
   );
 
+   const handleFatalError = React.useCallback(
+   (data) => {
+       console.log('inside handleFatalError ...')
+       postErrorLog(state.agentId, data)
+//       handleSubmit(data)
```

Comment:
How can I categorize the fatal errors from non fatal ones?

Reply:
as a temporary step you can trigger a `prompt()` and allow the user to choose? i don't like this because it can be a jarring experience for users, especially in the case of false positives, but it can be a good stopgap for now

Reply:
I already did this for type 2 errors. Thansk

Reply:
Do you mean do the `confirm()` inside `componentDidCatch()`? because I already do `confirm()` in the `window.onerror` event.